ludwig/schema/metadata/configs/common.yaml

```diff
@@ -0,0 +1,210 @@
+    \ method where during training, some neurons are randomly ignored or \u201C\
+    dropped out\u201D. Increasing dropout has the effect of making the training\
+    \ process more noisy and lowering overall network capacity, but it can\
+    \ be an effective regularization method to reduce overfitting and improve\
+    \ generalization."
+  example_value:
+    - 0.2
+  expected_impact: 3
+  literature_references:
+    - https://pytorch.org/docs/stable/generated/torch.nn.Dropout.html
+  suggested_values: 0.05 - 0.8
+  suggested_values_reasoning: Tuning dropout is really something to be done
+    when all of the big choices about architecture have been settled. Consider
+    starting with 0.5 and adjusting the dropout depending on observed model
+    performance.
+  ui_display_name: Dropout
+fc_layers:
+  default_value_reasoning: By default the stack is built by using num_fc_layers,
+    output_size, use_bias, weights_initializer, bias_initializer, norm, norm_params,
+    activation, dropout. When a list of dictionaries is provided, the stack
+    is built following the parameters of each dict for building each layer.
+  description_implications: The more layers that are specified the deeper and
+    higher capacity the model will be. This makes it possible to potentially
+    achieve better performance when a big anough amount of data is provided,
+    but also makes the model more computationally expensive and potentially
+    more prone to overfitting.
+  example_value:
+    - dropout: 0.1
+      output_size: 128
+    - norm: layer
+      output_size: 64
+  expected_impact: 1
+  related_parameters:
+    - output_size
+    - use_bias
+    - weights_initializer
+    - bias_initializer
+    - norm
+    - norm_params
+    - activation
+    - dropout
+  suggested_values_reasoning: It is easier to define a stack of fully connected
+    layers by just specifying num_fc_layers, output_size and the other individual
+    parameters. It will create a stack of layers with identical properties.
+    Use this parameter only if you need a fine grained level of control of
+    each individual layer in the stack.
+  ui_display_name: Fully Connected Layers
+flatten_inputs:
+  ui_display_name: null
+  expected_impact: 1
+norm:
+  default_value_reasoning: While batch normalization and layer normalization
+    usually lead to improvements, it can be useful to start with fewer bells
+    and whistles.
+  description_implications: Normalization helps stabilize the learning process
+    and can have a regularizing effect that can help with generalization.
+    It's often suggested that with normalization, you can use a higher learning
+    rate.
+  example_value:
+    - batch
+  expected_impact: 3
+  literature_references:
+    - https://machinelearningmastery.com/batch-normalization-for-training-of-deep-neural-networks/
+  related_parameters:
+    - norm_params
+  suggested_values: '"batch" or "layer"'
```

Comment:
Don't know if we should remove these necessarily, but I've always thought it a little unhelpful that the suggested values here are just both of the available values. I get that they are better than None, but I feel like from 85% of user's view they will see this and think that it's not very helpful since it's essentially suggesting that if the user is going to use this parameter they should use it. There are a couple other instances similar to this one throughout the metadata so do you think it would make sense to remove the suggested value here or just leave it in and gloss over it?

Reply:
Agreed, will remove.